<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpiralParametersSelectorClass</class>
 <widget class="QDialog" name="SpiralParametersSelectorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpiralParametersSelector</string>
  </property>
  <widget class="QLabel" name="EquationLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Equation: r = a*e^(b*u), start &lt;= u &lt;= end</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>156</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>121</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="A_label">
      <property name="text">
       <string>a</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="A_edit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="B_label">
      <property name="text">
       <string>b</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="B_edit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Start_label">
      <property name="text">
       <string>start</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="Start_edit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="End_label">
      <property name="text">
       <string>end</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="End_edit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SpiralParametersSelector.qrc"/>
 </resources>
 <connections/>
</ui>
